start/sw/web-app/about.md

```diff
@@ -1,10 +1,31 @@
 ---
-label: need-help
-
 title: Web Application
 subsection: web-app
 section: start-sw
-description:
+description: Short introduction to web application development
 ---
 
-# Web Application
+# Web Application Development
+
+Web application development is a broad topic, which covers range of different technologies. Under web application, we can imagine anything from classic server-side apps that render dynamic HTML pages, to real-time audio/video apps which allow us to watch movies or videochat online. Even if we only focus on the applications that users display in the web browser, a full-stack web developer needs to know dozens of technologies. What is common for all of them though, Fedora is here to offer all needed components and great development environment.
+
+For learning more about web technologies, visit for example web [w3schools.com](https://www.w3schools.com/).
```

Comment:
This should probably go to references (bottom) part of your document.

Reply:
Moved to the bottom.